src/VisualStudio/Core/Def/Implementation/ExtractInterface/ExtractInterfaceDialogViewModel.cs

```diff
@@ -17,6 +17,12 @@
 
 namespace Microsoft.VisualStudio.LanguageServices.Implementation.ExtractInterface
 {
+    internal enum InterfaceDestination
+    {
+        CurrentFile,
+        NewFile
+    };
```

Comment:
You already have the ExtractLocation enum.  Do we need this?

Reply:
Good point. Removing and making `ExtractLocation` into `ExtractInterfaceLocation`